### DISCUSSION TITLE
error: metadata-generation-failed

### DISCUSSION THREAD

**Discussion by Timothy-Rhodes** [coqui-ai/TTS#3705] [General Q&A] [answered]
Unfortunately, I'm still seeing the error as described here:
https://github.com/coqui-ai/TTS/issues/2986

This is on macOS. Can someone please let me know how to get the latest version or what I could be doing wrong? I'm just trying to "pip install TTS" but the metadata generation step fails every time.

**Comment by eginhard** [comment, marked as answer]
Not sure without the exact error message, but you could try installing [our fork](https://github.com/idiap/coqui-ai-TTS) (`pip install coqui-tts`) where I've removed some unnecessary dependencies and updated others.

**Reply by Timothy-Rhodes** [reply]
Thanks @eginhard. Installing the fork works for me.
